src/Analysers/TTbar_plus_X_analyser.cpp

```diff
@@ -95,6 +96,8 @@ void TTbar_plus_X_analyser::ePlusJetsSignalAnalysis(const EventPtr event) {
 		// BAT::TtbarHypothesis topHypothesis = hitFitAnalyserEPlusJetsRefSelection_->analyseAndReturn(event, jets, bJets, signalLepton );
 		// event->setTTbarHypothesis( topHypothesis );
 
+		likelihoodRecoAnalyserEPlusJetsRefSelection_->analyse(event, jets, bJets, signalLepton, MET_main);
+
 		// ref_selection_binned_HT_analyser_electron_->setScale(bjetWeight * efficiencyCorrection);
```

Comment:
Just noticed your new reco analyser is not run in the muon channel.  You should add it in under TTbar_plus_X_analyser::muPlusJetsSignalAnalysis